File: docs/multilogin-api/spec/code-samples/python/http_client/block_12.py
import http.client

conn = http.client.HTTPSConnection("launcher.mlx.yt", 45001)
payload = ''
headers = {
  'Accept': 'application/json'
}
conn.request("GET", "/api/v1/profile/stop/p/81b5627a-1212-4016-9467-3dbe4d6f78eb", payload, headers)
res = conn.getresponse()
data = res.read()
print(data.decode("utf-8"))

File: docs/multilogin-api/spec/code-samples/java/OkHttp/block_12.java
var myHeaders = new Headers();
myHeaders.append("Accept", "application/json");

var requestOptions = {
  method: 'GET',
  headers: myHeaders,
  redirect: 'follow'
};

fetch("https://launcher.mlx.yt:45001/api/v1/profile/stop/p/81b5627a-1212-4016-9467-3dbe4d6f78eb", requestOptions)
  .then(response => response.text())
  .then(result => console.log(result))
  .catch(error => console.log('error', error));
